packages/manager/modules/pci/src/index.scss

```diff
@@ -4,4 +4,12 @@
   @import '~bootstrap4/scss/_mixins';
   @import '~bootstrap4/scss/_utilities';
   @import '~bootstrap4/scss/_grid';
+
+  .minh-100 {
```

Comment:
Don't we have a utility CSS package like Bootstrap does ?

Reply:
Nope. There is `mh-100` but it's for `max-height` :-$
And for the class `overflow-auto`, it exists in bootstrap but in v4.3 ;-)

Reply:
Maybe we should have one then, so we don't have to rely on Boostrap ;)

Reply:
But bootstrap is great! :heart: :smile: